### PR TITLE
support oss bucket encryption configuration.

### PR DESCRIPTION
**Pull request by huiguangjun**

### CODE REVIEW

File: alicloud/resource_alicloud_oss_bucket_test.go [hunk 1]
```diff
@@ -331,6 +331,38 @@ func TestAccAlicloudOssBucketStorageClass(t *testing.T) {
...
+						"KMS"),
+					resource.TestCheckResourceAttr(
+						"alicloud_oss_bucket.sserule",
+						"server_side_encryption_rule.0.kms_master_key_id",
+						"11233455"),
+				),
+			},
+		},
+	})
+}
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_oss_bucket_test.go]: Please add a step to test updating server_side_encryption_rule.

File: website/docs/r/oss_bucket.html.markdown [hunk 2]
```diff
@@ -139,6 +151,7 @@ The following arguments are supported:
 * `lifecycle_rule` - (Optional) A configuration of [object lifecycle management](https://www.alibabacloud.com/help/doc-detail/31904.htm) (documented below).
 * `policy` - (Optional, Available in 1.41.0) Json format text of bucket policy [bucket policy management](https://www.alibabacloud.com/help/doc-detail/100680.htm) (documented below).
 * `storage_class` - (Optional, ForceNew) The [storage class](https://www.alibabacloud.com/help/doc-detail/51374.htm) to apply. Can be "Standard", "IA" and "Archive". Defaults to "Standard".
+* `server_side_encryption_rule` - (Optional) A configuration of server-side encryption (documented below).
```
Comment (xiaozhu36) [inline website/docs/r/oss_bucket.html.markdown]: Please add a tag for the `server_side_encryption_rule - (Optional, Available in 1.45.0+)`

File: alicloud/resource_alicloud_oss_bucket_test.go (hunk 1 above)
Comment (huiguangjun) [reply alicloud/resource_alicloud_oss_bucket_test.go]: done

File: website/docs/r/oss_bucket.html.markdown (hunk 2 above)
Comment (huiguangjun) [reply website/docs/r/oss_bucket.html.markdown]: done

**Pull request merged**